index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+
```

Comment:
It is a good practice to add a viewport meta tag. Kindly add the same to your code